apps/heft/src/utilities/CoreConfigFiles.ts

```diff
@@ -111,12 +111,42 @@ export class CoreConfigFiles {
       });
     }
 
-    const configurationFile: IHeftConfigurationJson =
-      await CoreConfigFiles._heftConfigFileLoader.loadConfigurationFileForProjectAsync(
+    let configurationFile: IHeftConfigurationJson;
+    try {
+      configurationFile = await CoreConfigFiles._heftConfigFileLoader.loadConfigurationFileForProjectAsync(
         terminal,
         projectPath,
         rigConfig
       );
+    } catch (e: unknown) {
+      if (!(e as Error).message.startsWith('Resolved configuration object does not match schema')) {
```

Comment:
We can't do an `instanceof` check or look for a `.code` property?

Reply:
No, this error doesn't have a .code property, it's just a normal error. I will however add an instanceof check